config/initializers/sidekiq.rb

```diff
@@ -7,6 +7,8 @@
     ssl: Settings.redis_ssl,
     network_timeout: 5
   }
+  config.options[:dead_timeout_in_seconds] = 2.weeks.to_i
```

Comment:
There does not seems to be docs around this besides the default value: https://github.com/mperham/sidekiq/blob/cea266a72f28b79b0f311552c18be62643893f7e/lib/sidekiq.rb#L39